products/sle/main.pm

```diff
@@ -570,7 +570,7 @@ sub load_reboot_tests() {
                 loadtest "installation/boot_into_snapshot";
             }
         }
-        if (get_var('ENCRYPT')) {
+        if (get_var('ENCRYPT') && !check_var('BACKEND', 'svirt')) {
```

Comment:
It's not obvious why svirt should be excluded. Can you comment? Or even better extract a "is…applicable" function